README.md

```diff
@@ -10,3 +10,14 @@ The webapp is set up with a Procfile. It simply requires `foreman` or `forego` t
 1. `bundle install`
 1. `foreman start`
 
+## Docker env
```

Comment:
Is this related to the fish shell completion?

Reply:
Not really, but because I had to run the env on Docker, so I thought I could update the Readme file for that.

Reply:
Ok. Updating the Readme is helpful too :)
It is possibly worth making it a separate pull request.

Reply:
Thanks @Insti.

@mhdali the docker change is completely separate, and while I'd like to accept the changes that relate to fish, I'm not so sure about the Docker instructions and would prefer to discuss those separately.